fastlane/lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -29,13 +37,32 @@ def self.run(params)
             File.write(path, new_plist)
           end
 
+          return plist if params[:map]
           return params[:value]
         rescue => ex
           UI.error(ex)
           UI.user_error!("Unable to set value to plist file at '#{path}'")
         end
       end
 
+      def self.deep_stringify(obj)
```

Comment:
Don't want to fight `plist` usage of string but I do want to support dev using `symbols`